c/tests/testlib.c

```diff
@@ -90,9 +90,9 @@ const char *paper_ex_sites = "1      0\n"
 const char *paper_ex_mutations = "0      2   1\n"
                                  "1      0   1\n"
                                  "2      5   1\n";
-/* Two (diploid) indivduals */
-const char *paper_ex_individuals = "0      0.2,1.5\n"
-                                   "0      0.0,0.0\n";
+/* Three (diploid) indivduals with an offspring */
```

Comment:
There's still only two and no offspring here, right?

Reply:
Changed accordingly

Reply:
Still says three?